### ISSUE TITLE
The keyboard focus order is incorrect for the grid table at the "Databases for assessment" page_A11y_[BM]SQL Migration extension for Azure Data Studio_Migrate to azure sql_Databases for assessment_Keyboard

### ISSUE THREAD

**Issue by Raj-ghuge** [microsoft/azuredatastudio#25903]
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification."**

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHub Tags:**
#A11yMAS; #A11yTCS; #Win11; #DesktopApp; #A11ySev2; #BM_SQL Migration extension for Azure Data Studio_Win32_AUG2024; #SQL Migration extension for Azure Data Studio; #Keyboard; #FocusOrder; #WCAG2.4.3; #AILimited; #FTP; #Win32;

## Environment Details:
Application Name: SQL Migration extension for Azure Data Studio
Microsoft Windows 11 version 23H2 (OS Build 22631.4037)
Version: 1.49.1 (user setup)
Commit: 2f696aa95a13ccabd7e0a4431d1d759e3a7b8cfd
Date: 2024-08-13T18:12:38.831Z
VS Code: 1.82.0
Electron: 25.9.8
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.22631
Azure SQL Migration v1.4.8

## Repro Steps:

1. Install Azure data studio and open.
2. Navigate by using TAB key till Extension press ENTER key and install "Azure SQL Migration"
3. Navigate by using TAB key till Respective server and right click.
4. Navigate by using TAB key till manage and press ENTER key.
5. Navigate by using TAB key till "Azure SQL Migration" and press ENTER key.
6. Navigate by using TAB key till migrate to azure sql and press ENTER key, saved session page will open.
7. Select start new session and give next, "Databases for assessment" page will open.
8. Navigate by using TAB key to the grid table in page and observe the issue.

**Note: Similar issue is observed for below scenario:**

**Issue 1: Migrate to azure sql_Database backup**

## Repro steps:

1. Install Azure data studio and open
3. Tab till Extension press enter key and install Azure SQL Migration
4. Tab till Respective server and right click
5. Tab till manage and press enter key
6. Tab till Azure SQL Migration and press enter key
7. Tab till migrate to azure sql and press enter key andSaved session page will open
8. Select start new session and give next ,Databases for assessment page will open
9. Select the respective databases and give next ,Assessment results and recommendations page will open
10. Select Azure SQL managed Instance/Virtual Machines from Azure SQL Target
11. Tab till Get Azure recommendation and press enter,
12. Get Azure Recommendation will open and select collect performance data now ,Browse folder and give start
13. Tab till view/select and press enter, select database and give next
14. Azure SQL Target will open and give next; Migration mode page will open.
15. Choose Select offline or online mode and give next
16. Database backup page will open
17. Select Azure blob container and give next
18. Azure Database Migration Service page will open and tab till Create, press enter
19. Create new Azure Database migration service page will open
20. Choose Select offline or online mode and give next
21. Database backup page will open.
22. Tab till grid table and observe the issue.

## Actual results:
The keyboard focus order is incorrect for the grid table at the "Databases for assessment" page as keyboard focus selected for one checkbox, at that time grid from the table also getting selected.
 
## Expected results:
The keyboard focus order should be correct for the grid table as keyboard focus selected for one checkbox, at that time grid from the table should not be selected.

## User Impact:
The Keyboard user might face difficulties and won't be able to perform action on the table.

## Attachments:
[25903_A11y_[BM]SQL Migration extension for Azure Data Studio_Migrate to azure sql_Databases for assessment_Keyboard.webm](https://github.com/user-attachments/assets/5a06fa09-37df-49d9-a9e0-224d289cf855)
[25903_Similar issue.webm](https://github.com/user-attachments/assets/3b0dcfdc-9b75-49db-aa2c-9e891f01c044)

**Comment by Samhitha66** [comment]
#Rev:samhithaks;

**Comment by akshika898** [comment]
As discussed with @Samhitha66 we have this issue that the row remaining selected even when the focus/tab moves out of table everywhere in ADS extensions and not specific to SQL Migration Extension.

**Comment by Raj-ghuge** [comment]
Verified the issue in the environment-
Version: 1.49.1 (user setup)
Commit: https://github.com/microsoft/azuredatastudio/commit/2f696aa95a13ccabd7e0a4431d1d759e3a7b8cfd
Date: 2024-08-13T18:12:38.831Z
VS Code: 1.82.0
Electron: 25.9.8
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.26100
The issue is fixed hence closing the bug.
#Regressed:11-12-24;
#Closed;
https://github.com/user-attachments/assets/0e7a8651-5ff4-4bfd-8e21-53653ce8617b
